draft-ietf-tls-esni.md

```diff
@@ -460,8 +460,8 @@ otherwise.
 
 To offer ECH, the client first chooses a suitable ECH configuration. To
 determine if a given `ECHConfig` is suitable, it checks that it supports the KEM
-algorithm identified by `ECHConfig.kem_id`, at least one KDF/AEAD algorithm
-identified by `ECHConfig.cipher_suites`, and the version of ECH indicated by
+algorithm identified by `ECHConfigContents.kem_id`, at least one KDF/AEAD algorithm
```

Comment:
In this section, it might be clearer to refer to `ECHConfig.contents.kem_id`, since `ECHConfig` has been defined, but `ECHConfigContents` only implicitly refers to `ECHConfig.contents`.

Reply:
Agreed, made this change for the whole "Offering ECH" section.